include/Surelog/Design/FileContent.h

```diff
@@ -68,15 +68,15 @@ class FileContent : public DesignComponent {
                    VObjectType type) const;  // Get first parent item of type
 
   NodeId sl_parent(
-      NodeId parent, const std::vector<VObjectType>& types,
+      NodeId parent, const std::unordered_set<VObjectType>& types,
```

Comment:
Don't mix two things in one PR: focus here is to change to the strong type not container choices.
So change this in a separate PR.

Reply:
PR #2945